idd/Energy+.idd.in

```diff
@@ -23824,6 +23824,40 @@ AirflowNetwork:MultiZone:WindPressureCoefficientValues,
       \units dimensionless
       \note Enter the WPC Value corresponding to the 36th wind direction.
 
+AirflowNetwork:ZoneControl:PressureController,
+	 \memo Define the Pressures control settings for a zone or list of zones.
+	 \memo If you use a ZoneList in the Zone or ZoneList name field then this definition applies
+	 \memo to all the zones in the ZoneList.
+ A1 , \field Name
+      \required-field
+      \reference AirflowNetworkZoneControlPressureControllerNames
+ A2 , \field Zone Name
+      \required-field
+      \type object-list
+      \object-list ZoneAndZoneListNames
+ A3 , \field Control Object Type
+      \required-field
+      \type choice
+      \key AirflowNetwork:MultiZone:Component:ZoneExhaustFan
+      \key AirflowNetwork:Distribution:Component:ReliefAirFlow
+      \note The current selection is AirflowNetwork:MultiZone:Component:ZoneExhaustFan 
+      \note or AirflowNetwork:Distribution:Component:ReliefAirFlow.
+ A4 , \field Control Name
+      \note Control names are names of individual control objects 
+      \required-field
+      \type object-list
+      \object-list ControlTypeNames
```

Comment:
Probably better to make this more specific to avoid confusion with HVAC control objects, maybe call it AirflowPressureControlTypeNames.